### DISCUSSION TITLE
Advice on wrong path at Repository URI form field

### DISCUSSION THREAD

**Discussion by matteopasqualin** [garethgeorge/backrest#190] [Ideas]
Hello all,

I would like to report I got an error when setting the path in Repository URI if I use the tilde sign ~/ to point to my home directory.
I am on Mac OS with Backrest version 0.16. The error/exception says: ... readonly filesystem. Indeed it is not.
So by setting the path this way: /Users/myname/myfolder/restic-repo Backrest works fine.

It could be that an advice on this issue in the form, would help users not to adopt tilde on path.

Maybe this issue is reported somewhere in docs, I didn't find it.

Thanks a lot and keep up with the great work!

**Comment by garethgeorge** [comment]
Hey, this is largely works as intended (I'd like to require that all paths in backrest's configs are absolute as it eliminates any ambiguity) but I'll aim to update the tooltip when I get a chance :)

**Reply by matteopasqualin** [reply]
Hello, ok I see. Well, the advice tooltip would be just fine. 
For the future, a more specific and explanatory exception wouldn't be bad either.
